### PR TITLE
Review page

### PR DESCRIPTION
**Pull request by adamsilver**
Review page now demonstrates how an incomplete section works when the user first goes to the page and when the user tries to proceed by pressing continue.

- Flash messages added
- Only course choices work just to demo how it works

### CODE REVIEW

File: app/assets/sass/components/_tag.scss [hunk 1]
```diff
@@ -32,11 +32,15 @@
 }
 
 // Section completion statuses
-.app-tag--unamended {
+.app-tag--unamended, {
```
Comment (fofr) [inline app/assets/sass/components/_tag.scss]: 👀

File: app/assets/sass/components/_tag.scss [hunk 2]
```diff
@@ -32,11 +32,15 @@
...
   color: govuk-colour("dark-grey");
 }
 
 .app-tag--submitted {
   @include tag("dark-grey", $filled: true);
 }
+
+.app-tag--not-complete {
+  @include tag("blue", $filled: false);
+}
```
Comment (fofr) [inline app/assets/sass/components/_tag.scss]: 👀

File: app/views/_includes/review/personal-statement.njk [hunk 3]
```diff
@@ -25,5 +25,28 @@
...
+      classes: 'govuk-button--secondary govuk-!-margin-bottom-0'
+    })}}
+
+  {% endset %}
+
+  {{ appSummaryCard({
+    attributes: {
+      id: 'personal-statement-error'
+    },
+    classes: "govuk-!-margin-bottom-6 app-summary-card--erro",
```
Comment (fofr) [inline app/views/_includes/review/personal-statement.njk]: `erro`

Comment (adamsilver) [reply app/views/_includes/review/personal-statement.njk]: Old code, reverting.

**Pull request merged**